api/TextTrack.json

```diff
@@ -542,54 +545,6 @@
           }
         }
       },
-      "oncuechange": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextTrack/oncuechange",
-          "support": {
-            "chrome": {
-              "version_added": "23"
-            },
-            "chrome_android": {
-              "version_added": "25"
-            },
-            "edge": {
-              "version_added": "12"
-            },
-            "firefox": {
-              "version_added": "31"
-            },
-            "firefox_android": {
-              "version_added": "31"
-            },
-            "ie": {
-              "version_added": "10"
-            },
-            "opera": {
-              "version_added": "≤12.1"
```

Comment:
On second review I see that the "oncuechange" and "cuechange_event" data doesn't match for Opera or Safari. There's probably a lot of small differences. Can you list what they are for ease of review?

Such differences are really the only thing worth looking at when reviewing these PRs, so maybe it's worth writing a script to list the differences and generate a boilerplate comment for each PR?